tests/build/build.go

```diff
@@ -498,13 +436,13 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build-ser
 			BeforeAll(func() {
 				fileToCreatePath := fmt.Sprintf(".tekton/%s-readme.md", customBranchComponentName)
 				switch gitProvider {
-				case "github":
+				case git.GitHubProvider:
```

Comment:
Was it possible to refactor this part of the code as well using `gitClient.CreateFile()`